projects/com.oracle.truffle.llvm.parser.impl/src/com/oracle/truffle/llvm/parser/impl/LLVMParserAsserts.java

```diff
@@ -40,4 +40,10 @@
         return objects;
     }
 
+    public static void assertNotNull(Object object) {
+        if (object == null) {
+            throw new AssertionError();
+        }
+    }
```

Comment:
I think having a central method is more flexible. As another example, we might want to selectively treat classes of errors (e.g., errors in the parser as in `LLVMParserAsserts`) differently. We already have an option `sulong.PerformanceWarningsAreFatal` where we either throw an exception or continue execution normally. We could follow a similar approach by using `LLVMParserAsserts` where we could control with an option whether we want to continue execution or throw an error.